tests/test_ops.py

```diff
@@ -0,0 +1,121 @@
+"""
+Unit tests for GPI2 operation and its utility functions.
+"""
+import pytest
+import torch
+import jax
+
+import pennylane as qml
+from pennylane import numpy as np
+import pennylane.math as math
+import tensorflow as tf
+from ionizer.ops import GPI
+
+jax.config.update("jax_enable_x64", True)
+
+interfaces = [
+        "autograd",
+        "torch",
+        "tf",
+        "jax",
+        "auto",
+        None
+    ]
+two_pi = 2 * np.pi
+
+def get_GPI_matrix(phi):
+    return np.array([[0, math.exp(-1j * phi)], [math.exp(1j * phi), 0]])
+
+
+class TestGPI:
+    @staticmethod
+    def circuit(phi): 
+        qml.Hadamard(wires=0)
+        GPI(phi, wires=0)
+        return qml.expval(qml.PauliZ(wires=0))
+    
+    @staticmethod
+    def circuit_unitary(phi): 
+        qml.Hadamard(wires=0)
+        gpi_matrix = get_GPI_matrix(phi)
+        qml.QubitUnitary(gpi_matrix, wires=0)
```

Comment:
This circuit and the test that uses it are testing the function defined in the test file rather than a library function :sweat_smile:

Reply:
Should be fixed